libraries/standard/http/src/http_client.c

```diff
@@ -1,6 +1,4 @@
-#include <assert.h>
```

Comment:
Why is this deleted? Where will this header exist?

Reply:
`http_client_internal.h`